### ISSUE TITLE
Requesting improvement for making custom PCB shapes

### ISSUE THREAD

**Issue by PickyBiker** [fritzing/fritzing-app#4005]
Making custom PCB shapes for Fritzing is a difficult task, this needs to be made much easier.

I would like to see the ability to import a .svg shape and have Fritzing automatically convert it to a PCB ready to use. If changes are necessary, then just redraw the .svg and import it again.

**Comment by KjellMorgenstern** [comment]
This is already pretty easy, you can load a custom shape by selecting a board, and then choosing to load an image file in the Inspector widget:

![image](https://user-images.githubusercontent.com/6494025/221404470-235f0c68-cb64-427a-979c-d6b4899f853c.png)

There are some requirements on that image of course. But without more information about what part you find difficult, or if you just struggled to find the "load image file" button, it is impossible to assess this issue.

**Comment by PickyBiker** [comment]
You are correct. I gave a poor description of the problem. It is not the loading of the image; it is the requirements of the .svg image using inkscape that is difficult. I guess it is really an inkscape issue.

**Comment by KjellMorgenstern** [comment]
So for example you would like to be able to load an SVG path like that:

https://yqnn.github.io/svg-path-editor/#P=M_10_10_L_100_10_L_55_90_Z_M_40_40_L_70_40_L_55_70_Z

And have it automatically used as a board by Fritzing?

**Comment by PickyBiker** [comment]
Yes, that would be great!
